scripts/elemental-airgap.sh

```diff
@@ -578,13 +581,11 @@ helm upgrade --create-namespace -n cattle-elemental-system --install elemental-o
 
 helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator $CHART_NAME_OPERATOR \\
   --set registryUrl=$LOCAL_REGISTRY \\
-  --set channel.repository=$CHANNEL_IMAGE_NAME
+  --set channel.image=$CHANNEL_IMAGE_NAME
 EOF
 }
 
 clean_up() {
-    is_hauler || return
```

Comment:
I just thought that is was to test for the binary and something that was forgotten. Maybe some comment can help here ;-) I will revert and add one.

And I have to do something else has it is the latest command run in the script and in that case it exit with a RC=1 so my new CI test fails because of this RC.

Reply:
Fixed!